modules/data.atmosphere/NAMESPACE

```diff
@@ -1,5 +1,8 @@
 # Generated by roxygen2: do not edit by hand
 
+S3method(predict,met)
+S3method(predict,subdaily.function)
+S3method(predict,subdaily.workflow)
```

Comment:
I'd recommend changing the names of these functions to use underscore rather than period so they export normally, rather than as S3method